packages/migrate/src/commands/DbPush.ts

```diff
@@ -124,12 +123,28 @@ You can now remove the ${chalk.red('--preview-feature')} flag.`)
         migrate.stop()
         throw e
       }
-      if (dbInfo.dbName && dbInfo.dbLocation) {
-        console.info(
-          `The ${dbInfo.dbType} ${dbInfo.schemaWord} "${dbInfo.dbName}" from "${dbInfo.dbLocation}" was successfully reset.`,
-        )
+      if (datasourceInfo.dbName && datasourceInfo.dbLocation) {
+        if (datasourceInfo.schema) {
+          console.info(
+            `The ${datasourceInfo.schema} schema of the ${datasourceInfo.prettyProvider} database from "${datasourceInfo.dbLocation}" was successfully reset.`,
+          )
+        } else if (datasourceInfo.prettyProvider === 'PostgreSQL') {
+          console.info(
+            `The ${datasourceInfo.prettyProvider} database "${datasourceInfo.dbName}" from "${datasourceInfo.dbLocation}" was successfully reset.`,
+          )
+        } else {
+          console.info(
+            `The ${datasourceInfo.prettyProvider} database "${datasourceInfo.dbName}" from "${datasourceInfo.dbLocation}" was successfully reset.`,
+          )
+        }
```

Comment:
Seems these are identical and the `datasourceInfo.prettyProvider === 'PostgreSQL` can be removed now.

Reply:
Indeed

Reply:
Improved